### ISSUE TITLE
DropDown MemberSelector property is not working on selected item

### ISSUE THREAD

**Issue by ArsenAghajanyan** [AvaloniaUI/Avalonia#2159]
Hello,

The DropDown MemberSelector property is working properly on the list, but it's not working on the selected item. See screenshots below`
![image](https://user-images.githubusercontent.com/33094487/49422648-1d22f300-f7ae-11e8-8001-6cb17fed2361.png)
![image](https://user-images.githubusercontent.com/33094487/49422658-2a3fe200-f7ae-11e8-96f4-c426d6322a78.png)
This is my .xaml code

`<DropDown MemberSelector="Name" Items="{Binding Items}" Width="200" Height="40" />`

and the code in the ViewModel file
```
    public class MainWindowViewModel : ViewModelBase
    {
        public List<Model> Items => new List<Model>
        {
            new Model
            {
                Name = "A",
                Number = 5
            },
            new Model
            {
                Name = "B",
                Number = 12
            }
        };
    }

    public class Model
    {
        public string Name { get; set; }
        public int Number { get; set; }
    }
```

Thank you!

**Comment by donandren** [comment]
this issues hsould be already fixed in master by #2122

**Comment by ArsenAghajanyan** [comment]
I'm not quite sure I know if it's fixed in master by #2122 or not. I was using the Alavonia project template, which I believe is using version 0.6.1.

**Comment by ArsenAghajanyan** [comment]
Thanks guys,
It works with the beta version build.
So, this is no longer a bug/issue.
